core/controllers/reader.py

```diff
@@ -310,11 +310,16 @@ def post(self, exploration_id):
         old_state_name = self.payload.get('old_state_name')
         # The reader's answer.
         answer = self.payload.get('answer')
+        # The answer handler (submit, click, etc.)
+        handler_name = self.payload.get('handler')
```

Comment:
Not yet. It will be removed as part of the refactor. I think we fixed it to be 'submit', though.